content/2017.04.getting-started-with-elm.rst

```diff
@@ -50,32 +47,42 @@ another layer to setup, configure and maintain in your stack.
 .. _Flow: https://flow.org/en/
 .. _TypeScript: https://www.typescriptlang.org/
 
-Elm brings functional programming to your browser, the language is
-statically typed so with Elm you won't see any run-time errors caused
-by wrong type. Being purely functional, Elm let you write code that is
-completely decoupled, it makes your code much more reusable and more
-easy to refactor.
+You can think of the Elm platform as the equivalent feature set of React_,
+Redux_, react-redux_, Babel_, eslint_, Flow/Typescript and Webpack_ all
+bundled in a single homogeneous package!
+
+.. _React: https://facebook.github.io/react/
+.. _react-redux: https://github.com/reactjs/react-redux
+.. _eslint: http://eslint.org/
+.. _Webpack: https://webpack.github.io/
+
+Elm brings functional programming to your browser. The language is
+statically typed, so with Elm you won't see any run-time errors caused
+by inconsistent typing. Being functional and pure, Elm let you write code
+that is completely decoupled, which makes your code more reusable and
+easier to refactor.
 
 
 How is Elm different?
 =====================
 
-Most of the time, using Elm_, the two lines it takes to load your app
-is all the JavaScript you will write. Your application will be written
-in Elm_ and compiled into JavaScript.
+Most of the time, using Elm, the two lines it takes to load your app
+from the HTML file is all the JavaScript you will write, as your Elm
+code will be transpiled to JavaScrit by the compiler.
```

Comment:
I would rather like using compile than transpiled. Because unlike CoffeeScript and Babel there is no map file and no need nor way to link the compiled JavaScript with the Elm code.